ExportCSV.ascx

```diff
@@ -1,34 +1,58 @@
 <%@ Control Language="C#" CodeBehind="ExportCSV.ascx.cs" Inherits="DotNetNuke.Modules.UserDefinedTable.ExportCsv" AutoEventWireup="false" %>
 <%@ Register TagPrefix="dnn" TagName="Label" Src="~/controls/LabelControl.ascx" %>
+<%@ Register TagPrefix="dnn" Namespace="DotNetNuke.Web.Client.ClientResourceManagement" Assembly="DotNetNuke.Web.Client" %>
+
+<dnn:DnnCssInclude runat="server" FilePath="~/Resources/Shared/components/TimePicker/Themes/jquery-ui.css" />
```

Comment:
Are you using jQueryUI for something specific? Does it fix a related bug or something?

Reply:
Good day. Im using jqueryUI css for calendar visualization. DNN don't include styles just scripts for JqueryUI. If this file is not included calendar is showed without any style.